open-api/paths/dataSetAttributeNames.yaml

```diff
@@ -32,5 +32,6 @@ get:
       #   - plot_keywords
       #   - title_year
     500:
```

Comment:
I don't think we have a specific ticket for rest api return codes. My tickets were focused on changing the query code itself to not throw an exception if no records matching the query are found. I will add a new ticket specific to the rest api since changing the backend code will effect how the rest api and that will affect the end consumer so we should be careful about when we release the change.